dbfit-java/core/src/main/java/dbfit/api/DbQuery.java

```diff
@@ -0,0 +1,32 @@
+package dbfit.api;
+
+import java.sql.ResultSet;
+import java.sql.SQLException;
+
+/**
+ * An object representing an executable database operation.
+ * Executing the operation returns a ResultSet result.
+ *
+ */
+public interface DbQuery extends AutoCloseable {
```

Comment:
As this is only used by `PreparedDbCommand` is it worth having this as an independent interface?